### ISSUE TITLE
7Span: Resources on Creating Accessible Web pages

### ISSUE THREAD

**Issue by WendyLBelcher** [PrincetonPEMM/pemm-issues#127]
### ALT Text
"Alternative (Alt) Text is meant to convey the “why” of the image as it relates to the content of a document or webpage. It is read aloud to users by screen reader software, and it is indexed by search engines. It also displays on the page if the image fails to load, as in this example of a missing image."

### Write Good Alt Text
- Add alt text all non-decorative images.
- Keep it short and descriptive, like a tweet.
- Don’t include “image of” or “photo of”.
- Leave alt text blank if the image is purely decorative
- It's not necessary to add text in the Title field. 

### Testing ALT
Use one of the [testing tools](https://accessibility.huit.harvard.edu/auto-tools-testing#free) to hide images and display their alt text. Does the page make sense with the images hidden, replaced by their text alternatives?

https://accessibility.huit.harvard.edu/describe-content-images#:~:text=Alternative%20(Alt)%20Text%20is%20meant,example%20of%20a%20missing%20image.

**Comment by WendyLBelcher** [comment]
### General Advice on Accessibility
Common errors
- Low contrast text
- Missing alternative text (ALT) for images
- Empty links (links with no meaningful text or image alt text, like a social media icon inside a link bloc)
- Missing form input labels (not applicable)
- Empty buttons (contains no text describing the function of the button)
- Missing document language (the page or post is not outputting the required language attribute on the HTML tag and assistive devices will not know what language the page is in)

https://webaim.org/projects/million/#errors
